### DISCUSSION TITLE
Unique Values

### DISCUSSION THREAD

**Discussion by taholmes160** [Budibase/budibase#8731] [Help]
HI Folks:

I am creating a fairly simple database -- Im using MySQL on the backend

the table that I am working on lists automobile manufacturers -- I want to ensure that there are no duplicates when adding a new manufacturer (Im dealing with salesmen here -- not IT Geeks) -- the column in the mysql table is set as unique -- and it wont allow it be non unique, but it fails kinda ugly -- (Pops up a red budibase error box) id like to make it fail a little more pretty -- is there an EASY (meaning no or very minimal coding) way to do this? (im a db guy, not a coder) 

Thanks Lots
TIM

**Comment by melohagan** [comment]
Hey @taholmes160 

Does this help? https://github.com/Budibase/budibase/discussions/6294#discussioncomment-2938345

**Comment by taholmes160** [comment]
it looks to me like it should -- I havent had time to test implement it
yet, but THANK YOU!!!!!
